SwrveGoogleSDK/src/main/java/com/swrve/sdk/gcm/SwrveGcmIntentService.java

```diff
@@ -8,19 +7,12 @@
 import android.os.Bundle;
 import android.support.v4.app.NotificationCompat;
 
-import com.google.android.gms.gcm.GoogleCloudMessaging;
-
-import java.util.Date;
+import com.google.android.gms.gcm.GcmListenerService;
 
 /**
  * Used internally to process push notifications inside for your app.
  */
-public class SwrveGcmIntentService extends IntentService implements ISwrveGcmService {
-
-    public SwrveGcmIntentService() {
-        super("SwrveGcmIntentService");
-    }
-
+public class SwrveGcmIntentService extends GcmListenerService implements ISwrveGcmService {
```

Comment:
I added the ISwrveGcmService to make it more testable, but I don't like it anymore. Think we should remove it.